saleor/tests/e2e/orders/utils/draft_order.py

```diff
@@ -0,0 +1,35 @@
+from saleor.graphql.tests.utils import get_graphql_content
+
+DRAFT_ORDER_CREATE_MUTATION = """
+mutation OrderDraftCreate($input: DraftOrderCreateInput!){
+  draftOrderCreate(input: $input){
+    errors{
+        message
+        field}
+    order{
+        id
+        }
+    }
+}
```

Comment:
Could you improve the formatting here?

Reply:
Hopefully it's better now: https://github.com/saleor/saleor/pull/13509/commits/fbeadf8530f053c2809676ffb411d6c1acc95fe2